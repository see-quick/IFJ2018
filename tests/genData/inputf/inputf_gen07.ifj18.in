5.5798
4.45
