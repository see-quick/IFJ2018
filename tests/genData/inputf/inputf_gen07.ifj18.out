0x1.651b71758e219p+2
